bin/fuzz.py

```diff
@@ -192,6 +195,21 @@ def run(self):
         self.tasks = 0
         self.queue = parallel.new_queue(lambda task: task.run(bar), pin=True)
 
+        def soft_exit(sig, frame):
+            if self.queue.aborted:
+                fatal('Running interrupted', force=True)
+            else:
+                self.queue.abort()
+                with bar:
+                    bar.clearline()
+                    message(
+                        'Running interrupted (waiting on remaining tasks)\n',
```

Comment:
add 'Press CTRL+C again to abort'? (that's what the `fatal` above does, right?)

Reply:
yes

Reply:
not sure if we should print that extra message though... pressing ctrl+c again is somewhat dangerous and can remove the `generator.yaml` completely (this is also the reason why I changed the code in the first place)

Reply:
I see. We could also print a warning for that. Or somehow disable the 2nd ctrl-c completely?

Reply:
i would not disable it if the program hangs for some reason.